api/src/controllers/assets.ts

```diff
@@ -60,11 +60,11 @@ router.get(
 
 		const transformation = pick(req.query, ASSET_TRANSFORM_QUERY_KEYS);
 
-		if (transformation.hasOwnProperty('key') && Object.keys(transformation).length > 1) {
+		if (Object.prototype.hasOwnProperty.call(transformation, 'key') && Object.keys(transformation).length > 1) {
```

Comment:
Not a fan of these.. Lets use `'key' in transformation` instead then

Reply:
Not a fan either from an aesthetic point of view... But it is the safest option, and I do like safe code 😃 
See https://stackoverflow.com/a/455366 and https://eslint.org/docs/rules/no-prototype-builtins

Reply:
Please tell me how you want to deal with this

Reply:
> Please tell me how you want to deal with this

I'd prefer using the `in` operator 👍🏻